app/Tools/Features.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Ushahidi\App\Tools;
+
+class Features
+{
+
+	/**
+	 * @param array $config
+	 */
+	public function __construct(array $config)
+	{
+		$this->config = $config;
```

Comment:
Right now this is direct from config. Future plans:
1. Handle defaults in the class, not config
2. Add some sort of environment based config so you can turn things on in dev/staging/prod easily, without db wrangling